Rating and Review

ks10825n opened the pull request:
This PR is to set up the API call for all reviews of the current product and misc file changes to mark where I want to render stuff. Let me know if any changes. Thanks!

kyelindholm reviewed (commented):
Do the changes to package.json and package-lock.json have to do with the changes to our testing in your previous PR? I don't want to solo-merge the last one because the test affects our project as a whole (so Leonardo should probably at least see it first), and I'm not sure if merging this PR to main without first merging the test changes would do something weird because of those previously mentioned files. 


Otherwise, all the code looks great!

ks10825n commented:
Just removed react-test-renderer! Sorry it was one of the native React libraries I threw in there trying to figure out a good way to test stuff (and like 5 other libraries that I removed). We don't need it for any testing we did previously together. For those, we should only need the testing-library/react and testing-library/jest-dom. And tbh, I think we can do the same things with Enzyme.

kyelindholm reviewed (approved):
Yeah, I think just switching to Enzyme/refactoring tests seems totally doable!
